src/custom/pages/Claim/ClaimsTable.tsx

```diff
@@ -0,0 +1,102 @@
+import { ClaimType, useClaimState } from 'state/claim/hooks'
```

Comment:
Moves table html code from index into this file, adds global claim state null checks

commented out is type code which i will fix in another PR - should be sdk types